src/main/resources/application.yml

```diff
@@ -56,7 +56,7 @@ workspace:
   sam.base-path: ${env.urls.sam}
 
   stairway-database:
-    force-clean: ${env.db.init}
+    force-clean: true
```

Comment:
You don't want to check this in if it does what I think it does. ⚠️

Reply:
!! it does, forgot to clean that up from personal env testing and I definitely don't want to check this in. thanks!